src/sort_cards.py

```diff
@@ -0,0 +1,39 @@
+def sort_cards(cards):
+    """Sort shuffled list of cards, sorted by rank.
+    
+    >>> sort_cards(['3', '9', 'A', '5', 'T', '8', '2', '4', 'Q', '7', 'J', '6', 'K'])
+    ['A', '2', '3', '4', '5', '6', '7', '8', '9', 'T', 'J', 'Q', 'K']
+    
+    """
```

Comment:
Nice solution.  I think you did a lot of extra work though.  You could use sorted, with your key parameter basically being your enum function.